sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/JniBlockMeta.kt

```diff
@@ -0,0 +1,9 @@
+package cash.z.ecc.android.sdk.internal.model
+
+internal data class JniBlockMeta(
```

Comment:
Should we drop `data` for this class?  I don't think we're using equals/hashcode/etc on this currently.